src/update.rs

```diff
@@ -88,9 +112,9 @@ impl Party0 {
         self,
         ChannelUpdateProposal {
             proposed_balance:
-                Balance {
-                    alice: balance_other,
-                    bob: balance_self,
+                GlobalBalance {
```

Comment:
What do you mean by "global"?

Reply:
Global may be the wrong term. It's an absolute view of the channel balance with respect to the update protocol i.e. identifying the sender and the receiver of the `ChannelUpdateProposal`.

It is global as opposed to a "local" view which would indicate balances for us and the counterparty.

I'll add documentation.

Reply:
I see. Maybe:
- `ChannelBalanceForProtocol` vs `ChannelBalance`
- `AbsoluteChannelBalance` vs `RelativeChannelBalance`

I don't think global/local is great because they both contain the same information, the labels are just different.